tests/unit/test_table.py

```diff
@@ -1711,31 +1713,21 @@ def test_callable_retry(self):
         self.assertEqual(client._table_data_client.mutate_rows.call_count, 2)
         self.assertEqual(result, expected_result)
 
-    def test_do_mutate_retryable_rows_empty_rows(self):
-        from google.cloud.bigtable_admin_v2.services.bigtable_table_admin import (
-            client as bigtable_table_admin,
-        )
-
-        table_api = mock.create_autospec(bigtable_table_admin.BigtableTableAdminClient)
-        credentials = _make_credentials()
-        client = self._make_client(
-            project="project-id", credentials=credentials, admin=True
-        )
-        client._table_admin_client = table_api
-        instance = client.instance(instance_id=self.INSTANCE_ID)
-        table = self._make_table(self.TABLE_ID, instance)
-
-        worker = self._make_worker(client, table.name, [])
-        statuses = worker._do_mutate_retryable_rows()
-
-        self.assertEqual(len(statuses), 0)
-
-    def test_do_mutate_retryable_rows(self):
+    def _do_mutate_retryable_rows_helper(
```

Comment:
This seems to be the same pattern `@pytest.mark.parametrize` seems to be trying to make possible. Would that be usable/useful here?

Reply:
`parametrize` is about pushing different data into the function, but a loses some of the documentation value (see the comments below in the functions which call the helper).  *Some* of the arguments here are really funky wires to control the flow, versus 'normal" data to pass to the method-under-test.

Reply:
Ok!